docs: link to ML.EVALUATE BQML page for score() methods

ashleyxuu opened the pull request:
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

ashleyxuu committed:
docs: link to ML.EVALUATE BQML page for score() methods

tswast commented on bigframes/ml/ensemble.py:
```diff
@@ -507,6 +507,11 @@ def score(
 
+        .. note::
+
+            We're using BigQuery ML.EVALUATE function (https://cloud.google.com/bigquery/docs/reference/standard-sql/bigqueryml-syntax-evaluate)
```
"We" doesn't make sense for our intended audience. Instead, something like "Output matches that of the BigQuery ML.EVALUTE function. See: LINK HERE for the outputs relevant to this model type."

Also, please link to the relevant output format, as that's what the user needs to know. That is to say:

https://cloud.google.com/bigquery/docs/reference/standard-sql/bigqueryml-syntax-evaluate#regression_models

or 

https://cloud.google.com/bigquery/docs/reference/standard-sql/bigqueryml-syntax-evaluate#classification_models

etc.

ashleyxuu committed:
Merge branch 'main' into b305767648-link-ml-EVALUATE-docs

milkshakeiii committed:
test: allow for alternative PCA solutions in tests (#143)

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://togithub.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

shobsi committed:
ci: Disable presubmit LLM tests temporarily (#144)

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://togithub.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

release-please[bot] committed:
chore(main): release 0.11.0 (#126)

Co-authored-by: release-please[bot] <55107282+release-please[bot]@users.noreply.github.com>

shobsi committed:
Revert "ci: Disable presubmit LLM tests temporarily (#144)" (#148)

This reverts commit 1641aff37d601b47e0bc4f25ff148be4f718bd1a, which was merged due to automerge label while still being discussed.

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://togithub.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

tswast committed:
refactor: make `to_pandas()` call `to_arrow()` and use local dtypes in DataFrame construction (#132)

Towards internal issue 280662868
🦕

ashleyxuu committed:
fix: address comment

ashleyxuu committed:
Merge branch 'main' into b305767648-link-ml-EVALUATE-docs

tswast reviewed (approved):
Love this. Thanks!

ashleyxuu committed:
Merge branch 'main' into b305767648-link-ml-EVALUATE-docs
